DependencyInjection/Configuration.php

```diff
@@ -254,7 +254,8 @@ private function addOdmSection(ArrayNodeDefinition $node)
                                 ->arrayNode('translation')
                                     ->addDefaultsIfNotSet()
                                     ->children()
-                                        ->scalarNode('alias')->defaultValue(Translation::LOCALE_NAMESPACE)->end()
+                                        ->scalarNode('alias')
+                                            ->defaultValue(class_exists(Translation) ? Translation::LOCALE_NAMESPACE : null)->end()
```

Comment:
ups, thanks for spotting this one. we are still supporting php 5.3 here so i use a string, not the ::class constant.